### PR TITLE
Feat(general): update governance to cw-1.0

### PR DESCRIPTION
**Pull request by maksk7**

### CODE REVIEW

**Review by ShadoySV — CHANGES_REQUESTED** [review]

File: scripts/migrate_permissions.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+async function main() {
+    const { terra, wallet } = newClient()
+    console.log(`chainID: ${terra.config.chainID} wallet: ${wallet.key.accAddress}`)
+    const network = readArtifact(terra.config.chainID)
+    console.log('network:', network)
+
+    // create propose new owner for our contracts
+    for (const key in network) {
```
Comment (epanchee) [inline scripts/migrate_permissions.ts:48]: We need to define each contracts manually as network.json contains not only contract addresses

Comment (P-Yevhenii) [reply scripts/migrate_permissions.ts:48]: Why? For example, if it is not the address of the contract, we will catch an error, skip this address and continue work. Also, we can specify a file that includes only the necessary contract addresses.

Comment (epanchee) [reply scripts/migrate_permissions.ts:48]: I don't see where we are catching errors in the current code

Comment (epanchee) [reply scripts/migrate_permissions.ts:48]: Ok, catching error is implicit but it should work. You are correct

**Review by epanchee — APPROVED** [review]

**Review by ShadoySV — APPROVED** [review]

**Pull request merged**